crates/miden-testing/src/kernel_tests/mod.rs

```diff
@@ -737,26 +737,40 @@ fn executed_transaction_output_notes() {
     // --------------------------------------------------------------------------------------------
     let output_notes = executed_transaction.output_notes();
 
-    // assert that the expected output note is present
+    // check the total number of notes
     // NOTE: the mock state already contains 3 output notes
     assert_eq!(output_notes.num_notes(), 6);
 
-    let output_note_id_3 = executed_transaction.output_notes().get_note(3).id();
-    let recipient_3 = Digest::from([Felt::new(0), Felt::new(1), Felt::new(2), Felt::new(3)]);
-    let note_assets_3 = NoteAssets::new(vec![combined_asset]).unwrap();
-    let expected_note_id_3 = NoteId::new(recipient_3, note_assets_3.commitment());
-    assert_eq!(output_note_id_3, expected_note_id_3);
+    // assert that the expected output note 1 is present
+    let output_note_1 = executed_transaction.output_notes().get_note(3);
+    let recipient_1 = Digest::from([Felt::new(0), Felt::new(1), Felt::new(2), Felt::new(3)]);
```

Comment:
```suggestion
    let recipient_1 = output_note_1.recipient_digest().expect("output note 1 should be a full note");
```

Would it make sense to fetch the recipient digest like this?

Reply:
Makes sense, thanks for explaining!